tests/jballerina-unit-test/src/test/resources/test-src/runtime/api/async/main.bal

```diff
@@ -18,6 +18,48 @@ import ballerina/test;
 import ballerina/jballerina.java;
 
 int globalVar = 2;
+
+public class ObjectMethodCallTest {
```

Comment:
The `ObjectMethodCallTest` name seems like a verb to me. Since it's a class, shall we name it as `ObjectMethodCallClass` or `ObjectMethodClass`?